images/linux/scripts/installers/validate-disk-space.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+################################################################################
+##  File:  validate-disk-space.sh
+##  Desc:  Validate free disk space
+################################################################################
+
+availableSpaceMB=$(( $(df / | sed 1d | awk '{ print $4 "\t" }')/1024 ))
```

Comment:
df supports -hm format that returns size in mb.

$ df / -hm
Filesystem     1M-blocks  Used Available Use% Mounted on
rootfs            120019 80502     39518  68% /

Reply:
Thanks, fixed.